docs/modules/reference/pages/daemons/daemon-config-files/trapd.adoc

```diff
@@ -3,6 +3,19 @@
 = Trapd
 
 Handles processing of xref:operation:events/sources/snmp-traps.adoc[SNMP traps] into events.
```

Comment:
When we updated the daemon section, we wanted to eliminate the verbiage for all the daemons. (Originally they were "The trapd daemon provides ..." or "The syslog daemon enables...". We can revisit this in light of the style guide, but for now, let's keep it as is so that all the daemon chapters have the same structure.

Reply:
Sounds good! I couldn't remember exactly what we'd decided, so I'll make a note of this.

Reply:
No problem. I think we updated the daemon section before you started ...